src/ts/components/subtitleoverlay.ts

```diff
@@ -121,11 +121,18 @@ export class SubtitleOverlay extends Container<ContainerConfig> {
       this.updateComponents();
     };
 
+    const clearInactiveSubtitles = () => {
+      const removedActiveSubtitles = subtitleManager.clearInactiveCues(player.getCurrentTime());
```

Comment:
NIT: For clarity, I would choose a name that reflects that these are actually `cues`

Reply:
Done in https://github.com/bitmovin/bitmovin-player-ui/pull/573/commits/bf0e75d23a90bb38d251c8c6ece7b18e82d07737